plutus-benchmark/marlowe/exe/PlutusBenchmark/Marlowe/Util.hs

```diff
@@ -25,18 +23,16 @@ module PlutusBenchmark.Marlowe.Util (
 , makeRedeemerMap
 , makeDatumMap
 , makeBuiltinData
-  -- * Rewriting
-, updateScriptHash
 ) where
 
 
 import Codec.Serialise (deserialise)
 import PlutusLedgerApi.V2 (Address (Address), BuiltinData, Credential (..), Datum (Datum),
                            DatumHash, LedgerBytes (getLedgerBytes),
                            OutputDatum (NoOutputDatum, OutputDatumHash), Redeemer (Redeemer),
-                           ScriptContext (..), ScriptHash, ScriptPurpose, TxId, TxInInfo (..),
-                           TxInfo (..), TxOut (..), TxOutRef (TxOutRef), Value, adaSymbol, adaToken,
-                           dataToBuiltinData, fromBuiltin, singleton)
+                           ScriptHash, ScriptPurpose, TxId, TxInInfo (..), TxOut (..),
```

Comment:
Will we need to change this to `PlutusLedgerApi.V3` at some point?  Is anything likely to change that will matter?

Reply:
Yes, I mentioned this a bit in the readme. The versioning is quite fixed atm and we have to manually update things if Marlowe changes unfortunately.